tools/realsense-viewer/realsense-viewer.cpp

```diff
@@ -262,6 +262,13 @@ int main(int argc, const char** argv) try
     viewer_model viewer_model;
     viewer_model.ctx = ctx;
 
+    rs2::log_to_callback( RS2_LOG_SEVERITY_INFO,
```

Comment:
Is info sufficiently high? Does this produce a lot of messages?

Reply:
Actually, I did this for testing and never removed -- it did not produce many messages

Reply:
Removed